### PR TITLE
API - User can see details of an ongoing task

### PR DESCRIPTION
**Pull request by JaimeCrz**
# [PT Story](https://www.pivotaltracker.com/story/show/172392525)

```
As a website user
in order to see my claimed tasks or help request I have created
I would like the site to show me details of the task
```
## Changes proposed in this pull request:
* Added a Profiles Controller
* Open new route for [ Index ] Profiles Controller
* Created index action for claimed tasks
* Restrict index action to Tasks with statuses claimed or delivered
* Restricts index action only to  those who log in / are authorized.

## What I have learned working on this feature:

* Refreshed knowledge of Rails controllers.
* Refreshed knowledge of  Routes & Restrictions.
* Usage of multiple conditions in .where statement.

### CODE REVIEW

File: app/controllers/api/v1/profiles_controller.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Api::V1::ProfilesController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    claimed_tasks = current_user.accepted_tasks.empty? ? "You don't have any claimed tasks." : current_user.accepted_tasks
+    created_tasks = current_user.tasks.empty? ? "You don't have any ongoing tasks." : current_user.tasks
```
Comment (blake-futchi) [inline app/controllers/api/v1/profiles_controller.rb:8]: You don't have any open requests (line 8) assuming these were created by the user in question. Claimed tasks I'm assuming are for the same user, correct?

**Review by blake-futchi — COMMENTED** [review]
overall looks pretty simple. Just have one clarification question

File: app/controllers/api/v1/profiles_controller.rb (hunk 1 above)
Comment (janko101) [reply app/controllers/api/v1/profiles_controller.rb:8]: We're sending two keys: Claimed and Created tasks. Inside Claimed tasks key you can see all the active tasks you claimed, if you didn't claim any it says the message that you don't have any claimed tasks. Inside Created tasks key it shows if you have any tasks you created to get help with if there isn't any you get the other message.

Comment (janko101) [reply app/controllers/api/v1/profiles_controller.rb:8]: So on a FE under User's profile you could have two buttons Created and Claimed where the user can view active tasks either created or confirmed

Comment (pierre-1) [reply app/controllers/api/v1/profiles_controller.rb:8]: Thanks for the explanation. Looks like we could now have a dashboard in the front end for each user.

**Comment by pierre-1** [comment]
Tutto bene. Good job on the testing!

**Review by robin-lillqvist — APPROVED** [review]
LGTM

**Pull request merged**